stingray/powerspectrum.py

```diff
@@ -549,6 +549,63 @@ def from_events(
             use_common_mean=use_common_mean,
         )
 
+    @staticmethod
+    def from_stingray_timeseries(
```

Comment:
same question as for Crossspectrum: does this enable the use of `Powerspectrum(ts1, ts2)` or does that still require only a light curve or event list?

Reply:
Here too, I preferred not to add the complication of a new argument to `self.__init__`. But we could in principle allow that too